inc/saiport.h

```diff
@@ -136,6 +136,9 @@ typedef enum _sai_port_media_type_t
 
     /** Media type copper. Remote advertise medium information as copper */
     SAI_PORT_MEDIA_TYPE_COPPER,
+
```

Comment:
This comment is applicable to all files.  The copyright statement is not up to date.  And I do not see Broadcom listed anywhere as a contributor.

Copyright (c) 2014 Microsoft Open Technologies, Inc.
 *
 *    Licensed under the Apache License, Version 2.0 (the "License"); you may
 *    not use this file except in compliance with the License. You may obtain
 *    a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
 *
 *    THIS CODE IS PROVIDED ON AN *AS IS* BASIS, WITHOUT WARRANTIES OR
 *    CONDITIONS OF ANY KIND, EITHER EXPRESS OR IMPLIED, INCLUDING WITHOUT
 *    LIMITATION ANY IMPLIED WARRANTIES OR CONDITIONS OF TITLE, FITNESS
 *    FOR A PARTICULAR PURPOSE, MERCHANTABILITY OR NON-INFRINGEMENT.
 *
 *    See the Apache Version 2.0 License for specific language governing
 *    permissions and limitations under the License.
 *
 *    Microsoft would like to thank the following companies for their review and
 *    assistance with these files: Intel Corporation, Mellanox Technologies Ltd,
 *    Dell Products, L.P., Facebook, Inc., Marvell International Ltd.

Reply:
@apogedik, broadcom has been a big contributor to the SAI, and we can certainly add broadcom. Since it is not specific to gear box. I suggest to open another PR to add broadcom name to all header files.

Reply:
Thanks.  Who creates PRs?  I am not familiar with the process.

Reply:
can you sync with Tushar?